gradle.properties

```
@@ -3,7 +3,7 @@ group=io.ballerina
 version=0.7.1-SNAPSHOT
 
 #dependency
-ballerinaLangVersion=2201.7.0
+ballerinaLangVersion=2201.7.0-20230620-082800-c0018aae
```

```suggestion
ballerinaLangVersion=2201.7.0
```

Since we don't need this as discussed offline.